base/abstractarray.jl

```diff
@@ -1132,9 +1132,9 @@ end
 """
     getindex(A, inds...)
 
-Return a subset of array `A` as specified by `inds`, where each `ind` may be an
-`Int`, an [`AbstractRange`](@ref), or a [`Vector`](@ref). See the manual section on
-[array indexing](@ref man-array-indexing) for details.
+Return a subset of array `A` as specified by `inds`, where each `ind` may be in
```

Comment:
This doesn't work gramatically. `ind` here is intended to refer to elements of the `inds` tuple, so the existing doc string is correct.

Reply:
It is not correct. I understand that `ind` refers to elements of the tuple, but the point is that `ind` can take many different values (not necessarily "an `Int`, an `AbstractRange`, or a `Vector`"). Here is  an example:
```
julia> x = [1 2; 3 4]
2×2 Array{Int64,2}:
 1  2
 3  4

julia> x[[1 2], view(y, 1), big(1)]
1×2 Array{Int64,2}:
 1  3
```
and as you can see:
1. the first index passed is a `Matrix` (not listed)
2. the second index passed is a 0-dimensional view (not listed)
3. the third index is `BigInt` (not listed)

There are so many types of values that are allowed that I think it is not worth to list them here, but that is why I propose to add "in particular" in this docstring.

The issue is that this docstring is confusing to the users, as they think that only the three types listed in it are allowed, while this is not the case.

Reply:
I think you want: "where each `ind` may be, for example,. ..."

Reply:
changed

(for my learning as I am not a native speaker: I thought that in this context "in particular" and "for example" are synonymous - what is the difference in their meaning here?)

Thank you!

Reply:
`in particular` is used for elaboration as in "Jupyter is an acronym. In particular, it was named after the languages Julia, Python and R" or specialization (to the exclusion of those not listed) as in "the Julia developers are really smart, Jeff Bezanson in particular" (implying that while everyone is smart, Jeff is even more so). However, it always needs to be preceded by a clause that it applies too, which is why the grammar didn't work here.